src/devices/technicmediumhubgyrosensor.ts

```diff
@@ -11,36 +11,30 @@ import * as Consts from "../consts";
 export class TechnicMediumHubGyroSensor extends Device {
 
     constructor (hub: IDeviceInterface, portId: number) {
-        super(hub, portId, ModeMap, Consts.DeviceType.TECHNIC_MEDIUM_HUB_GYRO_SENSOR);
+        const modes = [
+            {
+                name: "gyro",
+                input: true,
+                output: false,
+                raw: {min: -28571.419921875, max: 28571.419921875},
+                pct: {min: -100, max: 100},
+                si: {min: -2000, max: 2000, symbol: "DPS"},
+                values: {count: 3, type: Consts.ValueType.Int16}
+            }
```

Comment:
I dont understand how max of Int16 can be a decimal.

Reply:
the system communicates int16. rawMin and rawMax are not the boundaries of this values but the scaling baseline on the SI/pct min/max. That is the same as for the motors. their pos is a fully utilized int32 but rawMin/rawMax are -360 to 360.

Reply:
Oh, then I will remove the "clamping" logic of the `normalize` function.